config/config_python.py

```diff
@@ -0,0 +1,34 @@
+import pip
+import pkg_resources
+
+def main():
+    # Install gslab_tools if it's not installed yet or it has the wrong version
+    if not check_gslab_tools('4.1.0'):
+        pip.main(['install', 'git+http://git@github.com/gslab-econ/gslab_python.git@4.1.0'])
```

Comment:
We'll have to update this line for after the release. I think we can drop a tag in for the branch name [here](https://coderwall.com/p/-wbo5q/pip-install-a-specific-github-repo-tag-or-branch), which would help keep things reproducible.

Reply:
I agree.